### ISSUE TITLE
RFC: antenna stats

### ISSUE THREAD

**Issue by Schnobs** [KSP-RO/RealismOverhaul#695]
Here's what I have right now:
name - size - cone angle - energy/s - bandwidth - effective range:
KR-7      0.7m, 25°, 25kg, 0.05e, 10,  40Gm  scaled 0.5, for early comm satellites
HG55	 ~1.5m,  5°, 25kg, 0.07e,  2, 340Gm  Asteroid Day Antenna, looks vaguely Mariner-like
DTS-M1	   ??m, 35°  10kg, 0.05e, 15, 580Gm  ultimate short-range dish

pioneer   2.7m, 3.3°,  80kg, 0.12e,  1, 1500Gm (KR-14)
v'ger	  3.4m,   1°, 100kg, 0.16e, 10,  10Tm  (commtech1)

Comm88	  4.8m, 0.4°, 50kg, 0.03e, 25, 1500Gm stock folding dish scaled 2x
Gx128	  6.4m, 0.2°, 70kg, 0.08e,  15,  10Tm 

Orbits for scale:
Earth 150Gm
Mars 250Gm
Saturn 1400Gm

Power:
early probe core consumes 0.05
surveyor 0.16
38kg RTG provides 0.15 (pioneer carried three of these, voyager... about two)

So, in a nutshell: Mars with the first interplanetary dish requires some mission planning (opposition is out of range), similar but stricter for medium-ranged dishes and Saturn.

Bandwidth is pretty unimportant right now, but it doesn't need to stay that way. As a baseline, the Communotron-16 transmits 0.725data/second.

**Comment by Schnobs** [comment]
given that something along these lines has been merged, I don't think this issue serves any purpose anymore.
